core/src/mindustry/content/Blocks.java

```diff
@@ -1589,6 +1589,26 @@ public void drawBase(Tile tile){}
             reloadTime = 10f;
         }};
 
+        tsunami = new LiquidTurret("tsunami"){{
+            requirements(Category.turret, with(Items.metaglass, 100, Items.lead, 400, Items.titanium, 250, Items.thorium, 100));
+            ammo(
+                    Liquids.water, Bullets.waterShot,
+                    Liquids.slag, Bullets.slagShot,
+                    Liquids.cryofluid, Bullets.cryoShot,
+                    Liquids.oil, Bullets.oilShot
+            );
+            size = 3;
+            recoilAmount = 0f;
+            reloadTime = 1f;
```

Comment:
This won't work. The reload should be at least 2-3; turrets can only shoot once per frame, so any FPS below 60 would make this turret fire slower than it should.